web/html/src/components/table/TableDataHandler.tsx

```diff
@@ -341,10 +341,9 @@ export class TableDataHandler extends React.Component<Props, State> {
                   selectable={this.props.selectable}
                 >
                   {this.props.searchField}
-                  {this.props.additionalFilters &&
-                    this.props.additionalFilters.map((filter, i) => (
-                      <span key={"additional-filter-" + i}>{filter}&nbsp;</span>
-                    ))}
+                  {this.props.additionalFilters?.map((filter, i) => (
+                    <span key={"additional-filter-" + i}>{filter}&nbsp;</span>
```

Comment:
There is a small bug here, created a separate ticket for it: https://github.com/SUSE/spacewalk/issues/14072